layers/cmd/config/docgraph.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	"github.com/cloudprivacylabs/opencypher/graph"
+)
+
+func NewDocumentGraph() graph.Graph {
+	grph := ls.NewDocumentGraph()
+	// add indexes defined in config
+	cfg := GetConfig()
+	for nodeItr := grph.GetNodes(); nodeItr.Next(); {
```

Comment:
Iterate cfg.IndexProperties and call grph.AddNodePropertyIndex(prop) for each. They are not node properties.

Use grph.(*graph.OCGraph) to get to AddNodeProperties method